deposit_contract/tests/contracts/test_deposit.py

```diff
@@ -56,7 +56,7 @@ def deposit_input():
         (10, True),
         (55555, True),
         (2**64 - 1, True),
-        (2**64, False),
+        # (2**64, True),  # Note that all calls to `to_little_endian_64` have an input less than 2**64
```

Comment:
I'm not certain why you took the `assert` out and this test out. The method is exposed publicly and should thus have the assert blocker. The only way the assert should be removed is if the method is private, and the comment on max size passed in should stay within the method.

Reply:
> The only way the assert should be removed is if the method is private

This is another argument in favour of #1151.

> the comment on max size passed in should stay within the method

:+1:

Reply:
I'm in favor of it. The method was originally exposed publicly to be able to write tests directly against it because it was using weird vyper magic. I think it reasonable to make it private and have it tested implicitly by all the other tests that depend on it.

Reply:
@NIC619 or @hwwhww Can you help make `to_little_endian_64` private while keeping the tests working?

Reply:
The thing is we _won't_ be able to run tests directly against the method any more (unless we did weird dynamic surgery to the contract for testing.. no advised).

We just need to ensure that the methods we are testing that use the helper method produce the correct results. So making sure `get_deposit_count` and the merkle root (that calls the method on `amount`) work correctly tests the method enough.

Reply:
@JustinDrake Do you want to resolve this issue in this PR?